kuma/settings/common.py

```diff
@@ -953,6 +953,24 @@ def enable_candidate_languages():
         ),
         'output_filename': 'build/js/ace.js',
     },
+    'experiment-wiki-content': {
+        'source_filenames': (
+            'js/libs/mozilla.dnthelper.js',
+            'js/libs/mozilla.cookiehelper.js',
+            'js/libs/mozilla.trafficcop.js',
+            'js/experiment-wiki-content.js',
+        ),
+        'output_filename': 'build/js/experiment-wiki-content.js',
+    },
+    'experiment-framework-test': {
+        'source_filenames': (
+            'js/libs/mozilla.dnthelper.js',
+            'js/libs/mozilla.cookiehelper.js',
+            'js/libs/mozilla.trafficcop.js',
+            'js/experiment-framework-test.js',
+        ),
+        'output_filename': 'build/js/experiment-framework-test.js',
```

Comment:
@atopal asked for this as the first experiment - see [Phase 1](https://docs.google.com/document/d/1CUe6-EqNbpSkLMLxfd0RWW6-TbPZFRtLc0rMh2Uu3G0/edit#heading=h.l8azyzynxmrc)